test/test_files/abl_virtual_lidar/abl_virtual_lidar.inp

```diff
@@ -65,13 +65,23 @@ sampling.lidar.azimuth_table= 0 90.0
 sampling.lidar.elevation_table= 0 45.0
 
 sampling.spinner.type       = DTUSpinnerSampler
-sampling.spinner.dt_s = 0.1
-sampling.spinner.num_points = 25
-sampling.spinner.origin = 250.0 250.0 250.0
-sampling.spinner.length= 200.0
-sampling.spinner.time_table= 0 11.0
-sampling.spinner.azimuth_table= 0 90.0
-sampling.spinner.elevation_table= 0 300.0
+sampling.spinner.mode                       = fixed #use "hub" to connect to turbine named below 
+sampling.spinner.turbine                   = WTG01
```

Comment:
Does this work even though there is not turbine in the test case?
It would be best to not specify a turbine in this case.

Reply:
Yes, turbine is ignored if mode is not "hub"